sass/components/_mobile-menu.scss

```diff
@@ -0,0 +1,115 @@
+header{
+    position: relative;
+}
+.burger-menu {
+    z-index: 999;
+
+    @media screen and (max-width:1119px) {
+        position: absolute;
+        top: 0;
+        right: 0;
+        height: 100%;
+        background: #E17992;
+        width: 80%;
+        opacity: 1;
+        transition: opacity 250ms cubic-bezier(0.4, 0, 0.2, 1);
+        padding: 60px 21px;
+        @media screen and (min-width:450px) {
+            width: 320px;
+        }
+
+        &.is-hidden {
+            opacity: 0;
+            pointer-events: none;
+        }
+
+        .menu-desctop{
+            margin-top: 20px;
+            .item:last-child{
+                border-bottom:1px solid #E18298 ;
```

Comment:
color to variable , do not need space between color and semicolon (;)

Reply:
delete space